src/setting/setting.controller.ts

```diff
@@ -38,7 +38,7 @@ export class SettingController {
   @UsePipes(new ValidationPipe({ whitelist: true }))
   @Put("/changePhoneNumber")
   async changePhoneNumber(@Body() phoneDto: ChangePhoneNoDto, @Req() req, @Res() res): Promise<string> {
-    await this.settingService.changePhoneNumber(phoneDto, req.user.uid, req.user.token)
+    await this.settingService.changePhoneNumber(phoneDto, req.user.uid, phoneDto.token)
```

Comment:
just curious if phoneDto.token is contained in phoneDto, why do we need to pass in phoneDto.token separately?